test/e2e/placeholder_secrets_multi_case_test.go

```diff
@@ -273,6 +273,126 @@ spec:
+
+	yaml1 := `
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sg-test1
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sg-test2
+  annotations:
+    secretgen.carvel.dev/excluded-from-wildcard-matching: ""
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sg-test3
+  annotations:
+    secretgen.carvel.dev/excluded-from-wildcard-matching: ""
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-kappa
+  namespace: sg-test1
+type: kubernetes.io/dockerconfigjson
+stringData:
+  .dockerconfigjson: |
+    {
+      "auths": {
+        "www.sg-test1-server-a.com": {
+          "username": "sg-test1-secret-user-a",
+          "password": "sg-test1-secret-password-a",
+          "auth": "sgtest1-notbase64-a"
+        },
+        "www.sg-test1-server-b.com": {
+          "username": "sg-test1-secret-user-b",
+          "password": "sg-test1-secret-password-b",
+          "auth": "sgtest1-notbase64-b"
+        }
+      }
+    }
+---
+apiVersion: secretgen.carvel.dev/v1alpha1
+kind: SecretExport
+metadata:
+  name: secret-kappa
+  namespace: sg-test1
+spec:
+  toNamespaces:
+  - "*"
+  - sg-test3
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    secretgen.carvel.dev/image-pull-secret: ""
+  name: secret
+  namespace: sg-test2
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: "e30K"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    secretgen.carvel.dev/image-pull-secret: ""
+  name: secret
+  namespace: sg-test3
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: "e30K"
+---
+`
+	yaml1ExpectedContents := `{"auths":{"www.sg-test1-server-a.com":{"username":"sg-test1-secret-user-a","password":"sg-test1-secret-password-a","auth":"sgtest1-notbase64-a"},"www.sg-test1-server-b.com":{"username":"sg-test1-secret-user-b","password":"sg-test1-secret-password-b","auth":"sgtest1-notbase64-b"}}}`
+	emptyAuthsContents := `{"auths":{}}`
+
+	name := "test-placeholder-export-successful"
+	cleanUp := func() {
+		kapp.RunWithOpts([]string{"delete", "-a", name}, RunOpts{AllowError: true})
+	}
+
+	cleanUp()
+	defer cleanUp()
+
+	logger.Section("Initial Deploy", func() {
+		kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", name},
+			RunOpts{StdinReader: strings.NewReader(yaml1)})
+	})
+
+	// both ns have the exclusion annotation, but sg-test3 is also named explicitly.
+	nsToExpected := map[string]string{
+		"sg-test2": emptyAuthsContents,
+		"sg-test3": yaml1ExpectedContents,
+	}
+
+	logger.Section("Check placeholder secrets were populated appropriately", func() {
+		assertSecretsConvergeToExpected(t, nsToExpected, kubectl)
+	})
+
+	logger.Section("Removing annotation should populate secret", func() {
+		args := []string{"annotate", "namespace", "sg-test2", "secretgen.carvel.dev/excluded-from-wildcard-matching-"}
+		var out string
+		out, _ = kubectl.RunWithOpts(args, RunOpts{AllowError: false})
+		fmt.Println("Execution of kubectl", args, "  :", out)
```

Comment:
kubectl class already prints command it runs [0]. im not sure what the value of printing out is here. additionally RunOpts{AllowError: false} is unecessary. we should also probably just switch to `kubectl.Run(args)`

[0] https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/develop/test/e2e/kubectl.go#L31.